src/openms/source/ANALYSIS/ID/PeptideIndexing.cpp

```diff
@@ -56,17 +56,24 @@ using namespace std;
   const std::array<std::string, (Size)PeptideIndexing::MissingDecoy::SIZE_OF_MISSING_DECOY> PeptideIndexing::names_of_missing_decoy = { "error" , "warn" , "silent" };
 
 
-  // internal datastructure to store match information (not exported)
+  // internal data structure to store match information (not exported)
   struct PeptideProteinMatchInformation
   {
-    OpenMS::Size protein_index; //< index of the protein the peptide is contained in
-    OpenMS::Int position; //< the position of the peptide in the protein
+    Hit::T peptide_index; ///< index of the peptide
```

Comment:
```suggestion
    Hit::T peptide_index; //< index of the peptide
```

Reply:
documenting members is done with `///< ....`, so I guess it would be better to fix the other two members as well?
https://www.doxygen.nl/manual/docblocks.html#memberdoc